Doc/library/imaplib.rst

```diff
@@ -30,12 +30,14 @@ Three classes are provided by the :mod:`imaplib` module, :class:`IMAP4` is the
 base class:
 
 
-.. class:: IMAP4(host='', port=IMAP4_PORT)
+.. class:: IMAP4(host='', port=IMAP4_PORT[, timeout])
```

Comment:
I would specify this as `[, timeout=None]`. It annoys me when default values are hidden: it makes it impossible for me to write a wrapper that uses the same default value.